Nobar/Scenes/Main/Component/GuideCVC.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  GuideCVC.swift
+//  Nobar
+//
+//  Created by 이유진 on 2022/07/14.
+//
+
+import UIKit
+
+import Then
+import SnapKit
+
+final class GuideCVC: UICollectionViewCell {
+  
+  private let thumbnailImageView = UIImageView()
+  
+  private let titleLabel = UILabel().then {
+    $0.textColor = Color.black.getColor()
+    $0.font = Pretendard.size13.bold()
+    $0.lineBreakMode = .byWordWrapping
+    $0.numberOfLines = 2
+  }
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    render()
+    configUI()
+  }
+
+  @available(*, unavailable)
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func prepareForReuse() {
+    titleLabel.text = ""
+  }
+}
+
+// MARK: - UI & Layout
+
+extension GuideCVC {
+
+  private func render() {
+    addSubviews([
+    thumbnailImageView,
+    titleLabel
+    ])
+    thumbnailImageView.snp.makeConstraints {
+      $0.top.leading.trailing.equalToSuperview().inset(12)
+      $0.height.equalTo(74)
+      
+    }
+    titleLabel.snp.makeConstraints{
+      $0.top.equalTo(thumbnailImageView.snp.bottom).offset(8)
+      $0.leading.trailing.equalToSuperview().inset(12)
+    }
+  }
+
+  private func configUI() {
+    backgroundColor = Color.gray01.getColor()
+    layer.cornerRadius = 10
+    layer.applyShadow(color: UIColor(red: 0, green: 0, blue: 0, alpha: 1), alpha: 0.05, x: 1, y: 1, blur: 2, spread: 0)
+  }
+  
+ func setData(data: GuideModel){
```

Comment:
현재는 외부에서 사용할 때 data라는 필드를 두번 사용하게 되는데요!
언어 중 Swift만 제공하고 있는 argument label을 응용하면 조금 더 스위프트스럽게 메소드 네이밍을 해서 개선할 수 있어요~
요 방법을 제안을 드립니다!

```suggestion
 func setData(_ data: GuideModel)
```
-> `cell.setData(model)`

```suggestion
 func setData(with data: GuideModel)
```

-> `cell.setData(with: model)`

요렇게 외부에서 열리기 때문에 조금 더 영어 읽듯이 사용할 수 있게 됩니다!

Reply:
호우! 센스있는 리뷰 감사합니당~~ 스위프트스러운 인간이 되보도록하겠슴다!!

Reply:
네~~ 이 스위프트 유진 하세요~~